elements/pfelement/reveal.js

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
@mwcz did we intend to remove reveal.js ?

Reply:
Yes, it's a compiled asset that shouldn't have been there to begin with.  @castastrophe noticed that it was a stowaway. :)

Reply:
Oh and it isn't gone for good, it lives in `elements/pfelement/src/reveal.js`.  This just removed the compiled version.

Reply:
Listen, Gandalf Taps Musically  👍